docs/usage/faq.md

```diff
@@ -15,10 +15,34 @@ Two possible solutions are,
 In any case, files need to be served with a web server and cannot be loaded from
 local file system.
 
+## How can I download external files from within Pyodide?
+
+To download a file from the internet to the [emscripten virtual file system](https://emscripten.org/docs/api_reference/Filesystem-API.html), we currently recommend using `from js import fetch`. A complete file downloader and converter is described below:
+```python
+from js import fetch
+
+async def js_fetch(url, path):
+    response = await fetch(url)
+    js_buffer = await response.arrayBuffer()
+    py_buffer = js_buffer.to_py()  # this is a memoryview
+    stream = py_buffer.tobytes()   # now we have a bytes object
+
+    with open(path, "wb") as fh:
+        fh.write(stream)
```

Comment:
Perhaps we should wait to merge this until #1865 is merged, because then the code can be made simpler (and we can copy the data one fewer time)?

Reply:
sure that makes sense